source/casts.ts

```diff
@@ -1,20 +1,21 @@
-import { ExternalId, FileLink, Sequence, SequenceListScope, SequenceRowsRetrieve } from "@cognite/sdk";
```

Comment:
Reformatting and type improvements. This will be deprecated in the next version.